lib/topic_query.rb

```diff
@@ -165,7 +165,8 @@ def list_latest
 
   def list_read
     create_list(:read, unordered: true) do |topics|
-      topics.order('COALESCE(tu.last_visited_at, topics.bumped_at) DESC')
+      topics.where('tu.last_visited_at IS NOT NULL')
+        .order('COALESCE(tu.last_visited_at, topics.bumped_at) DESC')
```

Comment:
If `tu.last_visited_at IS NOT NULL` then the `COALESCE` here isn't doing anything and can be removed.

I would also add a spec to make sure it works properly.